addons/website_event/static/src/snippets/s_events/000.scss

```diff
@@ -20,17 +15,12 @@
             }
 
             .s_events_event_cover {
-                min-height: 100%;
+                max-height: 410px;
```

Comment:
We can indeed use rem instead of px in both places

Concerning a variable, I'm not sure, since the content of cards can differ, we might want to use different values 🤔

Reply:
My concern was avoiding using the same or similar magic numbers in several places, but if design-wise we don't care that they differ :shrug: